01-import_and_maxfilter.py

```diff
@@ -208,6 +208,22 @@ def load_data(bids_basename):
                         extra_params=extra_params,
                         kind=config.get_kind())
 
+    # Select only the channel types we wish to analyze.
+    if config.get_kind() == 'meg' and ('mag' in config.ch_types or
+                                       'grad' in config.ch_types):
+        picks = mne.pick_types(raw.info, eog=True, ecg=True)
+
+        if 'mag' in config.ch_types:
+            picks += mne.pick_types(raw.info, meg='mag')
+        if 'grad' in config.ch_types:
+            picks += mne.pick_types(raw.info, meg='grad')
+    elif config.get_kind() == 'meg':
+        picks = mne.pick_types(raw.info, meg=True, eog=True, ecg=True)
+    else:
+        picks = mne.pick_types(raw.info, eeg=True, eog=True, ecg=True)
```

Comment:
I realize that if you want to analyze grad or mag maybe you still want to use the full MEG sensor array in maxfilter. So I would pick mag or grad after calling maxfilter.

can you check that maxfilter works uniformly if you select grad or mag only on the sample data?

Reply:
Excellent point, I never thought about that! I will move the picking after Maxfilter. Do you still want me to run the test you proposed, anyway?